contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -110,9 +110,11 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
         // Pausing is a very serious situation - we revert to sound the alarms
         checkProtocolPauseState();
         checkActionPauseState(vToken, Action.BORROW);
-
         ensureListed(markets[vToken]);
 
+        uint256 borrowCap = borrowCaps[vToken];
+        require(borrowCap != 0, "market borrow cap is 0");
```

Comment:
Is this an optimization to save gas? Otherwise, I think this check is already done later (line 134)